src/main/java/DilemmaDetector/Simulator/CollisionDetector.java

```diff
@@ -46,12 +46,12 @@ public boolean detectOutOfRoad(Actor mainVehicle){
         int lastLaneLeft =  scenarioModel.getLanes().get(Model.Side.LEFT).entrySet().size() + 1;
         int lastLaneRight = scenarioModel.getLanes().get(Model.Side.RIGHT).entrySet().size() + 1;
 
-        double leftBorderY = lastLaneLeft * RigidBodyMapper.LANE_WIDTH * (-1);
-        double rightBorderY = lastLaneRight * RigidBodyMapper.LANE_WIDTH;
+        double leftBorderY = lastLaneLeft * RigidBodyMapper.LANE_WIDTH;
+        double rightBorderY = lastLaneRight * RigidBodyMapper.LANE_WIDTH * (-1);
 
```

Comment:
Faktycznie to chyba ja pomyliłem right z left

Reply:
I chyba w RigidBodyMapperze trzeba zmienić tak samo

Reply:
Zmieniłem. Na tyle proste te generowane scenariusze są w GUI, że się nawet nie zorientowałem, że jest źle